nipype/algorithms/modelgen.py

```diff
@@ -175,7 +176,11 @@ def bids_gen_info(bids_event_files,
         with open(bids_event_file) as f:
             f_events = csv.DictReader(f, skipinitialspace=True, delimiter='\t')
```

Comment:
This should resolve the Python 2/Unicode issue.

```suggestion
            f_events = csv.DictReader(f, skipinitialspace=True, delimiter=str('\t'))
```